11_install_agent/80_agent_configuration.txt

```diff
@@ -2,13 +2,12 @@
 
 ==== Configure Rudder Agent
 
-Configure the IP address of the Rudder Root Server in the following file
+Configure the IP address or hotname of the Rudder Root Server in the following file
 
 ----
 
-sudo tee /var/rudder/cfengine-community/policy_server.dat <<EOF
-@@replace_by_rudder_server_ip@@
-EOF 
+echo '<rudder server ip or hostname>' > /var/rudder/cfengine-community/policy_server.dat
+
```

Comment:
Please document here that by default the agent will look for a server named "rudder" in the configured resolver domains. So if you installed your Rudder server as "rudder.company.com", you don't need to edit this file at all.